### PR TITLE
Make build work on windows (have to manually download stockfish exe)

### PR DESCRIPTION
**Pull request by DanielAlbl**

### CODE REVIEW

File: build_common/src/lib.rs [hunk 1]
```diff
@@ -1 +1,3 @@
 pub mod command_config;
+pub mod helpers;
+pub use crate::helpers::*;
```
Comment (Dillon-Roller) [inline build_common/src/lib.rs:3]: `helpers` can be private and you can remove `crate`

Comment (Dillon-Roller) [reply build_common/src/lib.rs:3]: the structure is `mod` at top, and then a space, and then `use`

File: Dockerfile.fly [hunk 2]
```diff
@@ -3,5 +3,5 @@
 FROM ghcr.io/oxideops/chess:main
 WORKDIR /app
 COPY . .
-RUN cargo build -p server
-ENTRYPOINT ["./target/debug/server"]
+RUN cargo build -p server --release
```
Comment (Dillon-Roller) [inline Dockerfile.fly:6]: How come we aren't using dx build?

File: build_common/src/helpers.rs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    );
+    let mut args = vec!["build".into()];
+    if env::var("PROFILE") == Ok("release".into()) {
+        args.push("--release".into());
+    }
+
+    vec![CommandConfig {
+        program: PathBuf::from("trunk"),
+        args,
+        dir: Some(PathBuf::from(get_project_root())),
```
Comment (Dillon-Roller) [inline build_common/src/helpers.rs:76]: can remove `PathBuff::from` here

File: build_common/src/lib.rs [hunk 4]
```diff
@@ -1,2 +1,4 @@
 pub mod command_config;
```
Comment (Dillon-Roller) [inline build_common/src/lib.rs:1]: command_config and helpers can be private

File: build_common/src/helpers.rs [hunk 5]
```diff
@@ -0,0 +1,79 @@
+use crate::CommandConfig;
+use std::env;
+use std::path::PathBuf;
+
```
Comment (Dillon-Roller) [inline build_common/src/helpers.rs:4]: Should we put this in here?

```
const NODE_MODULES_BIN: &str = "node_modules/.bin";
const STYLES_DIR: &str = "styles";
const INPUT_CSS: &str = "input.css";
const OUTPUT_CSS: &str = "output.css";
```

File: build_common/src/lib.rs (hunk 4 above)
Comment (DanielAlbl) [reply build_common/src/lib.rs:1]: I don't believe it

**Pull request merged**